OSDOCS CQA MCO-1: MCO Core Configuration and Customization II (#113871)
openshift/openshift-docs: modules/rhcos-load-firmware-blobs.adoc

  name: 98-worker-firmware-blob
storage:
  files:
  - path: /var/lib/firmware/<package_name>
    contents:
      local: <package_name>
    mode: 0644
openshift:
  kernel_arguments:
    - 'firmware_class.path=/var/lib/firmware'
----
+
where:

`storage.files.path`:: Specifies the path on the node where the firmware package is copied to.

> `storage.files.contents.local`:: Specifies a file with contents that are read from a local file directory on the system running Butane. The path of the local file is relative to a `files-dir` directory, which must be specified by using the `--files-dir` option with Butane in the following step.

Reviewer: In the event step changes occur maybe "subsequent step" would be more open?

> `storage.files.mode`:: Specifies the permissions for the file on the {op-system} node. It is recommended to set `0644` permissions.

Reviewer: ```suggestion
`storage.files.mode`:: Specifies the permissions for the file on the {op-system} node. Red{nbsp}Hat recommends to set `0644` permissions.
```

"it" is a vague referent.
